apps/redi-talent-pool/src/react-query/use-tpjobseekercv-mutation.ts

```diff
@@ -1,5 +1,24 @@
 import { useMutation, useQueryClient } from 'react-query'
-import { updateCurrentUserTpJobseekerCv } from '../services/api/api'
+import {
+  createCurrentUserTpJobseekerCv,
+  updateCurrentUserTpJobseekerCv,
+  deleteCurrentUserTpJobseekerCv,
+} from '../services/api/api'
+
+export function useTpjobseekerCvCreateMutation() {
```

Comment:
Nice keeping them all in one file. I recently wrote some code where I split the three CrUD functions into three files, but that's unnecessary bloat. This is better.